backend/kegstarter/votingbooth/permissions.py

```
@@ -1,11 +1,11 @@
 from rest_framework.permissions import BasePermission
 
+from kegstarter.votingbooth.models import Vote, Poll
+
 
 class PollIsNotClosed(BasePermission):
     def has_object_permission(self, request, view, obj):
-        if hasattr(obj, 'closed'):
+        if isinstance(obj, Poll):
             return not obj.closed
-        elif hasattr(obj, 'poll'):
+        elif isinstance(obj, Vote):
             return not obj.poll.closed
-        else:
-            raise AttributeError('{} is not associated with a poll.'.format(obj))
```

The idea was to fail early with an informative message for anyone who tried to set this permission on objects that don't have polls. Not technically necessary to make things work, I just thought it'd be helpful during future development